localization/replace/english/wc_buildings_l_english.yml

```diff
@@ -1,22 +1,22 @@
 ﻿l_english:
   
- building_type_elephant_pens_01:1 "Elekkantry"
- building_type_elephant_pens_01_desc:1 "Elekks are kept and trained here. They make excellent beasts of burden, useful for both work and war."
- building_elephant_pens_01:1 "Elekk Pens"
- building_elephant_pens_01_desc:1 "These sturdy stables can keep even the most unruly elekk secured."
- building_elephant_pens_02:1 "Elekk Traps"
- building_elephant_pens_02_desc:1 "New elekks are collected by trapping them using various means, such as pits or snares. Sometimes a female elekk will even be used as bait."
- building_elephant_pens_03:1 "Elekk Training Grounds"
- building_elephant_pens_03_desc:1 "Elekks are not easy beasts to teach, but in the training grounds, the most experienced trainers do their utmost to provide the crown with beasts of war."
- building_elephant_pens_04:1 "Elekk Riding Grounds"
- building_elephant_pens_04_desc:1 "In the riding grounds, elekks are put to the test, carrying both crew, armor, and equipment."
+ building_type_elephant_pens_01:1 "Beastiary"
+ building_type_elephant_pens_01_desc:1 "Gargantuan beasts are kept and trained here. They make excellent beasts of burden, useful for both work and war."
+ building_elephant_pens_01:1 "Wide Pens"
+ building_elephant_pens_01_desc:1 "These sturdy stables can keep even the most unruly beasts secured."
+ building_elephant_pens_02:1 "Steel Traps"
+ building_elephant_pens_02_desc:1 "New beasts are collected by trapping them using various means, such as pits or snares. Sometimes a female will even be used as bait."
+ building_elephant_pens_03:1 "Spacious Training Grounds"
+ building_elephant_pens_03_desc:1 "Behemoths are not easy beasts to teach, but in the training grounds, the most experienced trainers do their utmost to provide the crown with beasts of war."
```

Comment:
Shouldn't they be called (gargantuan) beasts here, too? The switch to Behemoths is a little confusing.